### DISCUSSION TITLE
Add Teams as a dropdown field for new github issues

### DISCUSSION THREAD

**Discussion by bhubbard** [community/community#6824] [Projects and Issues]
Hey guys, loving the new issues, while I know I can create a new dropdown field type and manually populate it, it would be great if you can have the option for org teams as a dropdown field. I can see this expanded in the future for workflows (eg notifications).

Maybe give the option when adding it to be a single select vs multiple.

**Comment by timoyates** [comment]
Hi @bhubbard, thanks for the feedback! This makes a lot of sense. We're looking into new fields all the time and I'll be taking your suggestion back to the team. Cheers!
